docs/PREACT_BADGES.md

```diff
@@ -0,0 +1,328 @@
+## Badges
+
+Badges are self-configured in the Searchspring Management Console.
+
+To displays badges the Result card must include the [OverlayBadge](https://searchspring.github.io/snap/#/components-preact?params=%3Fpath%3D%2Fstory%2Fmolecules-overlaybadge--default) and [CalloutBadge](https://searchspring.github.io/snap/#/components-preact?params=%3Fpath%3D%2Fstory%2Fmolecules-calloutbadge--default) components. 
+
+
+### OverlayBadge
+
+The `OverlayBadge` component wraps elements (children) that should have badges overlayed - typically the product image.
+
+```jsx
+<OverlayBadge controller={controller} result={controller.store.results[0]}>
+	<img src='/images/example.png'/>
+</OverlayBadge>
+```
+
+### CalloutBadge
+
+The `CalloutBadge` component displays badges inline and can be placed in any position in the Result card. 
+
+```jsx
+<CalloutBadge result={controller.store.results[0]} />
+```
+
```

Comment:
I think before we go into custom badges we should mention the supported "out of the box" components available, and maybe link to the storybook for each.